### PR TITLE
Implement to get user id and profile, Fix #55

### LINKED ISSUE #55: Add getGroupMemberIds / getRoomMemberIds API
ref:

https://devdocs.line.me/en/#get-group-room-member-ids
It would be nice the APIs take care of the continuationToken part and return a plain array.

### PR DESCRIPTION
**Pull request by hirohisa**
- https://devdocs.line.me/en/#get-group-room-member-profile
- https://devdocs.line.me/en/#get-group-room-member-ids

### CODE REVIEW

**Comment by tokuhirom** [comment]
https://github.com/line/line-bot-sdk-go/pull/54
https://github.com/line/line-bot-sdk-java/pull/126
https://github.com/line/line-bot-sdk-python/pull/53
https://github.com/line/line-bot-sdk-nodejs/pull/15

Other library uses method name like `get_group_member_profile`.
Could you change the method names like other language's SDKs.

**Comment by hirohisa** [comment]
@tokuhirom , Thank you. I modified method's name.

**Comment by tokuhirom** [comment]
CI is failing :P

File: lib/line/bot/client.rb [hunk 1]
```diff
@@ -167,14 +167,62 @@ def get_message_content(identifier)
...
+      # Get user IDs of a group
+      #
+      # @param group_id [String] Group's identifier
+      # @param continuation_token [String] Identifier to return next page
+      #                                   (next property to be included in the response)
+      #
+      # @return [Net::HTTPResponse]
+      def get_group_member_ids(group_id, continuation_token=nil)
+        endpoint_path  = "/bot/group/#{group_id}/members/ids"
+        endpoint_path += "?start=#{continuation_token}" unless continuation_token.nil?
```
Comment (kimoto) [inline lib/line/bot/client.rb]: unless object.nil? という表現がわかりにくい気がするので、
if object で良さそうな気がしました。

get_room_member_ids のところも同様です

Comment (kimoto) [inline lib/line/bot/client.rb]: unless object.nil? だと意図がわかりにくい気がします
if object のほうがわかりやすい気がしました

Comment (hirohisa) [reply lib/line/bot/client.rb]: 👍

**Comment by hirohisa** [comment]
Thanks for review.

**Pull request merged**